en/docs/publish/api-microgateway/quick-start-guide.md

```diff
@@ -0,0 +1,96 @@
+# Quick Start Guide
+
+## Design Your First API
+
+This section is a step-by-step guide to create, publish, and invoke an API in the WSO2 API Microgateway.
+
+### Before you begin...
+
+1. Install [docker](https://docs.docker.com/engine/install/ubuntu/).
+2. Install the [docker-compose](https://docs.docker.com/compose/install/).
+
+### Objectives
+1. Setup Microgateway and CLI tool (APICTL).
+2. Create and deploy an API project.
+3. Invoke the API using a generated key.
+
+ Let's get started...
+ 
+### Step 1 - Setup Microgateway and CLI tool(APICTL)
+
+1.  Download the CLI tool(APICTL) and the microgateway distribution from the 
+    [github release page's](https://github.com/wso2/product-microgateway/releases) Assets and 
+    extract them to a folder of your choice.
+    
+    From here onwards, CLI tool extracted location will be referred as `CLI_HOME` and Microgateway distribution extracted 
+    location would be referred as `MG_HOME`.
+
+2.  Using your command line client tool add the 'CLI_HOME' folder to your PATH variable.
+
+    ``` bash
+    export PATH=$PATH:<CLI_HOME>
+    ```
+
+### Step 2 - Create and deploy an API project
+
+1.  Let's create our first project with name "petstore" by adding the 
+    [open API definition](https://petstore.swagger.io/v2/swagger.json) of the petstore . 
+    You can do that by executing the following command using your command line tool.
+    
+    ``` bash
+    apictl init petstore --oas https://petstore.swagger.io/v2/swagger.json
+    ```
+
+    The project is now initialized. You should notice a directory with name "petstore" being created in the location 
+    where you executed the command. 
+
+2.  Now lets start the microgateway on docker by executing the docker compose script inside the `MG_HOME`. 
+    Navigate to `MG_HOME` and execute the following command.
+
+    ``` bash
+    docker-compose up -d
+    ```
+    
+    Once containers are up and running, we can monitor the status of the containers using the following command
+    
+    ``` bash
+    docker ps | grep mg-
+    ```
+
+3.  Now let's deploy our first API to Microgateway using the project created in the step 3. 
+    Navigate to the location where the petstore project was initialized.
+    Execute the following command to deploy the API in the microgateway.
+       
+    ``` bash
+    apictl mg deploy --host https://localhost:9843 --file petstore  -u admin -p admin -k
+    ```
+    
+    !!! note  
+        The user credentials can be configured in the configurations of the `MG_HOME` distribution. 
+        `admin:admin` is the default accepted credentials by the microgateway adapter.
+        Go to `MG_HOME/resources/conf/config.toml` and modify as below.
+        
+        ``` toml
+        [server]
```

Comment:
This config should be an array right, @VirajSalaka since muliple developers should be able to deploy APIs

Reply:
yeah will change that immediately

Reply:
https://github.com/wso2/product-microgateway/pull/1547
@AmaliMatharaarachchi please change the content accordingly as per above mentioned PR.

Reply:
Done